### PR TITLE
Add banner and page on pages transition

### PR DESCRIPTION
**Pull request by apburnes**
Part of https://github.com/18F/federalist/issues/3436

Proposed changes in this pull request:
- [Add banner to site](https://federalist-a44a7672-afb1-473e-acb2-067f644c0967.app.cloud.gov/preview/18f/federalist.18f.gov/apb/pages-banner/)
- [Add page with announcement](https://federalist-a44a7672-afb1-473e-acb2-067f644c0967.app.cloud.gov/preview/18f/federalist.18f.gov/apb/pages-banner/cloud-gov-pages/)

### CODE REVIEW

**Review by davemcorwin — APPROVED** [review]
Thanks @apburnes that blue background is for real! A little disappointed that there aren't more emojis...

Does the content need to be reproduced verbatim? It seems a little repetitive.

On a serious note, the cloud.gov icon is a bit low, `vertical-align: text-bottom;` looks a bit better to me...

**Comment by apburnes** [comment]
@davemcorwin not sure on the content. Let's throw out the idea to the business team.

Also, took me like 10 tries to see the diff on the `text-bottom` vs `middle` but it does work just a bit better.

**Comment by apburnes** [comment]
Reworked the language on the announcement a bit. Let me know what you think.

**Review by davemcorwin — APPROVED** [review]
:shipit:

**Comment by amirbey** [comment]
@apburnes - attempting some wordsmithing ...

File: pages/cloud-gov-pages.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      </div>
+      <hr class="hr-light">
+      <p class="pages-prose">
+        We are undergoing a transition to be more aligned with our cloud.gov suite of services. Federalist is changing to
+        <img
+          alt="cloud.gov logo icon"
+          src="{{site.baseurl}}/assets/images/logos/cg-logo-icon.svg"
+        />
+        <span>cloud.gov Pages</span>!
+        Pages is the same great services provided to you with your current Federalist agreement, just with a new name and some new features to coming soon. We are working on a transition guide and FAQ, and will start publishing updates at the end of <span>June, 2021</span>.
```
Comment (amirbey) [inline pages/cloud-gov-pages.md:30]:
```suggestion
        Pages offers the same great services provided to you with your current Federalist agreement, just with a new name along with some new features soon to come. We are working on a transition guide and FAQ, and will start publishing updates at the end of <span>June, 2021</span>.
```

**Review by amirbey — COMMENTED** [review]
@apburnes  - LGTM .... I added a rewording suggestion

**Pull request merged**